docs/concepts/composition.md

```diff
@@ -328,6 +352,8 @@ spec:
     # object can therefore have unexpected consquences and is not recommended.
     # Instead patch specific annotations by specifying their keys.
     - fromFieldPath: metadata.annotations[example.org/app-name]
+  - name: external-name
```

Comment:
Moved this to a separate patchset so the example XR/XRC's would still work properly.